scripts/lunch.coffee

```diff
@@ -0,0 +1,3 @@
+module.exports = (robot) ->
+    robot.hear /What are we having for lunch?/i, (res) ->
+        res.send("/poll Where should we go for lunch? :gbk: :pizza: :spaghetti: :curry: :sushi: :taco: :burrito: :ramen: :chicken:")
```

Comment:
@jhg14 Would be good to have it documented in the robot help maybe? I let you have the wow effect tomorrow first ;-)